src/pyrokinetics/local_geometry/metric.py

```diff
@@ -142,15 +142,7 @@ def __init__(self, local_geometry: LocalGeometry, ntheta=None, theta=None):
 
         # This defines the reference magnetic field as B0:
         # dpsidr / (B0 * a) = <Jacobian * g^zetazeta> * (R0 / a) / q
-        bref_units = local_geometry.dpsidr.units / local_geometry.rho.units
-
-        # Needs to explicitly be 1 * B0 regardless of units
-        if "Bunit" in str(bref_units):
-            bref_units *= 1.0 / local_geometry.bunit_over_b0
-        elif "B0" not in str(bref_units):
-            raise ValueError("Need to convert to a standard normalisation first")
-
-        self.dpsidr = self.Y * local_geometry.Rmaj / self.q * bref_units
+        self.dpsidr = self.Y * local_geometry.Rmaj / self.q * local_geometry.B0
```

Comment:
Brackets around  mid-statement division to improve clarity of intent?

Reply:
Do you mean?

`self.dpsidr = self.Y * (local_geometry.Rmaj / self.q) * local_geometry.B0`

Reply:
Changed!